packages/flutter_tools/lib/src/isolated/native_assets/macos/native_assets_host.dart

```diff
@@ -99,13 +99,13 @@ Future<void> setInstallNamesDylib(
   String newInstallName,
   Map<String, String> oldToNewInstallNames,
 ) async {
-   final ProcessResult setInstallNamesResult = await globals.processManager.run(
+  final ProcessResult setInstallNamesResult = await globals.processManager.run(
     <String>[
       'install_name_tool',
       '-id',
       newInstallName,
-      for (final MapEntry<String, String> entry in oldToNewInstallNames.entries)
-        ...<String>['-change', entry.key, entry.value],
+      for (final MapEntry<String, String> entry in oldToNewInstallNames
```

Comment:
Ubernit: consider using an object destructuring pattern to pull out the key and value.

Reply:
I think that makes it even more verbose, because of lints that require explicit type annotations (then for both a key and a value variable).

As this is just refactoring of existing code, I hope it's ok if I leave it like this.

Reply:
Yup, that's no problem, just figured I'd mention it. I forgot about those lints...